.github/CODEOWNERS

```diff
@@ -43,8 +53,7 @@
 /sdk/appconfiguration/                                               @xiangyan99 @YalinLi0312
 
 # PRLabel: %App Configuration Provider
-/sdk/appconfiguration/azure-appconfiguration-provider/
-@mametcal @albertofori @avanigupta @mrm9084
```

Comment:
@jsquire @mrm9084 this path was completely broken.

Reply:
Was? I am unable to see a difference between this and the change.

Reply:
Also the path looks correct.

Reply:
Other than the newline at the end of what was 46, these look identical to me.

Reply:
That is what was broken you cannot split the owners and the paths on separate lines.